src/libs/workspace-utils.ts

```diff
@@ -161,26 +210,23 @@ export const regionConstraintMessage = (workspace: BaseWorkspace): string | unde
   const regions = getRegionConstraintLabels(workspace.policies);
   return regions.length === 0
     ? undefined
-    : `Workspace storage and compute resources must remain in the following region(s): ${regions.join(', ')}.`;
+    : `Workspace storage and compute resources must remain in the following ${pluralize(
+        'region',
+        regions.length
+      )}: ${regions.join(', ')}.`;
 };
 
 const isGroupConstraintPolicy = (policy: WorkspacePolicy): boolean => {
   return policy.namespace === 'terra' && policy.name === 'group-constraint';
 };
 
 /**
- * Returns true if the workspace has any data access controls (group constraint policies).
+ * Returns true if the workspace has any group constraint policies (data access controls).
  */
-export const hasDataAccessControls = (workspace: WorkspaceWrapper): boolean => {
+export const containsGroupConstraintPolicy = (workspace: WorkspaceWrapper): boolean => {
```

Comment:
Renaming for consistency with other policy methods, where we favor the actual name of the policy instead of the user-visible "label".

Reply:
I was really hoping to go the other way and make code and UI more consistent.

Also, naming nit... "workspace has policy" seems more natural vs "workspace contains policy".

Reply:
Also on the subject of consistency...
- protected data is named "contains" and takes a list of policies
- region constraint is named "has" and takes a workspace
- group constraint is named "contains" and takes a workspace

Reply:
`containsProtectedDataPolicy` is only being used for eventing, which is tricky because we don't always have a complete workspace object. It perhaps should be updated though so that it has the ability to event "hasProtectedData" for the GCP case as well. But for now, since it is only looking at the policy array, I left it as "contains".

For group-constraint and region-constraint, which takes a workspace, I standardized on "has".

As far as UI vs. code names, I prefer to stick with the name of the actual policies because those won't change… the UI labels certainly may change.

Reply:
"Contains" reads well there since it takes a list of policies.

My reasoning for wanting code and UI to match is so that you don't have to remember the mapping to find the code relevant to some UI.

Reply:
But my thought is that I will *know* the TPS policy names (from the server response), and it's easier to find the related terra-ui code if the names match the policy names. I had trouble remembering that "data access controls" meant "group-constraint".